src/app/addjson/_components/textArea/TextArea.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import React, { useState } from 'react';
+import { useRouter } from 'next/navigation';
+
+const TextArea: React.FC = () => {
+  const router = useRouter();
+  const [jsonInput, setJsonInput] = useState<string>('');
+  const [error, setError] = useState<string | null>(null);
+  const [showDetailedError, setShowDetailedError] = useState<boolean>(false);
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const value = e.target.value;
+    setJsonInput(value);
+    setError(null);
+  };
+
+  const handleSubmit = () => {
+    try {
+      const jsonData = JSON.parse(jsonInput);
+      if (jsonData && typeof jsonData === 'object') {
+        router.push('/success');
+      } else {
+        throw new Error('Input is not a valid JSON object');
+      }
+    } catch (error: any) {
```

Comment:
don't use any type

Reply:
I tried using 'Error' type but I got an error saying that I can only use "any" or "unknown" type.

Reply:
```
catch (error) {
        if (error instanceof Error) {
          setError(error.message);
        } else {
          setError("An unknown error occurred");
        }
    }
```
try this solution to solve the problem